Changelog.md

```diff
@@ -1,5 +1,8 @@
 # FOSSA CLI Changelog
 
+## v3.8.37
```

Comment:
[Prefer] that we bump the minor version, as I believe this change constitutes a significant enough modification in what constitutes a successful analysis.

Prior to this, the convention was that Fossa would exit fatally if it could not find anything to report. This was to avoid scenarios where someone installs `fossa analyze && fossa test`, and CI would pass, but no "test" was being conducted (as the CLI was not discovering anything). This sometimes led to the incorrect assumption that code is compliant, when in reality FOSSA couldn't detect anything (while customer believing that we found dependencies, and found no issues).

Reply:
That's a good point. I'm open to that , let me post in the group channel to get the others' thoughts as well.